### DISCUSSION TITLE
live or meeting 实时字幕

### DISCUSSION THREAD

**Discussion by Takenforgranted** [immersive-translate/immersive-translate#1965] [Ideas]
### 新功能 | New Feature

1.能够对一些实时视频（如直播和会议等）进行语音的识别记录、翻译，为其生成双语字幕。

### 为什么您想要添加这些功能 | Why you want to add these features

1.在昨天尝试观看油管上的邦TV的时候突然发现沉浸式翻译无法实现实时翻译字幕的功能
2.在探索过程中发现已经有很多别的插件已经实现了该项功能
3.如果为了字幕多加装一个插件不够简洁而且容易出现冲突

**Comment by theowenyoung** [comment]
youtube 直播的字幕目前已经支持。

另外 zoom, google meet, teams 网页端的在线会议已经支持双语字幕。
